Resources/Localization/zh-Hant.lproj/Localizable.strings

```diff
@@ -3311,3 +3311,23 @@
 
 /* Info alert when editing a generated column */
 "The current field \"%@\" is a generated column and therefore cannot be edited." = "The current field \"%@\" is a generated column and therefore cannot be edited.";
+
+/* Pin a view to the top of the list of tables
```

Comment:
Next time these can be added only to `en` strings and build will take care of translating and copying to all other languages :)

Reply:
Got it. Am using JetBrains AppCode instead of xcode because of my familiarity with intelliJ IDEA and all the IDE features and shortcuts. The IDE created the strings in all language files for me so I let it be that way.